src/Common/src/Interop/Windows/kernelbase/Interop.OpenCommPort.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class KernelBase
+    {
+#pragma warning disable BCL0015 // Disable Pinvoke analyzer errors, needed until buildtools can be updated to remove kernel32.dll!OpenCommPort
```

Comment:
I doubt we'll ever get to this unless we actually ship the next version of UWP from this release branch in the future.